src/duv/udp.c

```diff
@@ -1 +1,173 @@
 #include "udp.h"
+
+duk_ret_t duv_new_udp(duk_context *ctx) {
+  uv_udp_t *udp = duk_push_fixed_buffer(ctx, sizeof(uv_udp_t));
+  duv_check(ctx, uv_udp_init(duv_loop(ctx), udp));
+  duv_setup_handle(ctx, (uv_handle_t*)udp, DUV_UDP);
+  return 1;
+}
+
+duk_ret_t duv_udp_broadcast(duk_context *ctx) {
+  dschema_check(ctx, (const duv_schema_entry[]) {
+    {"enable", duk_is_boolean},
+    {0,0}
+  });
+  uv_udp_t *udp = duv_require_this_handle(ctx, DUV_UDP_MASK);
+   
+  duv_check(ctx, uv_udp_set_broadcast(udp, duk_get_int(ctx, 1)));
+  return 0;
+}
+
+static void duv_push_sockaddr(duk_context *ctx, struct sockaddr_storage* address, int addrlen) {
+  char ip[INET6_ADDRSTRLEN];
+  int port = 0;
+  return;
+  if (address->ss_family == AF_INET) {
+    struct sockaddr_in* addrin = (struct sockaddr_in*)address;
+    uv_inet_ntop(AF_INET, &(addrin->sin_addr), ip, addrlen);
```

Comment:
code taken from tcp.c, should be fixed there by author and taken over or even better used by both modules